test/integration/visitor_can_add_items_to_cart_test.rb

```diff
@@ -104,4 +104,50 @@ class VisitorCanAddItemsToCart < ActionDispatch::IntegrationTest
     assert page.has_content?("Super yummy")
     assert page.has_content?("Total: $20")
   end
+
+  test "user can adjust the quantity of an item in the cart" do
+    SET UP A CART WITH SOMEHTING IN IT
+    visit "/cart"
+
+    within ('.slotachips') do
```

Comment:
`(...)` interpreted as grouped expression.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.